src/etc/config.xml.sample

```diff
@@ -93,18 +93,19 @@
       <track6-prefix-id>0</track6-prefix-id>
     </lan>
   </interfaces>
-  <dhcpd>
-    <lan>
-      <enable/>
-      <range>
-        <from>192.168.1.100</from>
-        <to>192.168.1.199</to>
-      </range>
-    </lan>
-  </dhcpd>
   <unbound>
     <enable>1</enable>
   </unbound>
+  <dnsmasq>
+      <enable>1</enable>
+      <port>0</port>
+      <interface>lan</interface>
```

Comment:
if people don't assign a "lan" interface in console assignment this can be a problem. we also need to make sure migrations do the normalize dance then on this field and the one in dhcp_ranges ;(

Reply:
@fichtner but isn't this the same for isc at the moment?

https://github.com/opnsense/core/blob/3e299b29b9d0b1aaa17d1f950a6ef62676541c40/src/etc/config.xml.sample#L96-L104

In practice this likely means we need to replace the dhcpd sections in console.inc, which I haven't looked at yet (but should do indeed)

Reply:
dhcp is disabled when lan is not used, yes, this ties back to the console.inc madness

as far as isc doing it, also yes, but it's not an mvc migration-breaks-on-validation case

Reply:
the console part should be handled implemented via https://github.com/opnsense/core/pull/8378/commits/70b1672e8dc64e52c7cbe1f9c3266006bc194a97 now.